### DISCUSSION TITLE
Roadmap for editable DataGrid

### DISCUSSION THREAD

**Discussion by nstratton-srx** [microsoft/fluentui#33412] [General]
I can't see any references to the DataGrid in your Fluent Roadmap. Are there any discussion around making DataGrid an editable grid? I understand that's a sizeable undertaking. 

What we're ultimately looking for is a compact grid. The editable grids in Dataverse are huge space gobbling grids. We pay for a compact grid and would appreciate an editable grid from MS.

**Comment by ling1726** [comment]
@nstratton-srx could you please create a feature ask for us? https://github.com/microsoft/fluentui/issues/new/choose

We don't have any plans for first class support for editable grids today, since we haven't really gotten an ask to do so. We'll need a nice feature request and see how it prioritizes on our backlog
